assets/src/edit-story/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -108,16 +107,39 @@ function PageAttachmentPanel() {
   );
 
   const isDefault = _ctaText === defaultCTA;
+  const showTextInput = Boolean(url) && !isInvalidUrl;
+
+  const handleChange = useCallback(
+    ({ target }) => {
+      const { value } = target;
+      // This allows smooth input value change without any lag.
+      _setCtaText(value);
+      debouncedCTAUpdate(value);
+    },
+    [debouncedCTAUpdate]
+  );
+
+  const handleBlur = useCallback(
+    ({ target }) => {
+      if (!target.value) {
+        updatePageAttachment({ ctaText: defaultCTA });
+        _setCtaText(defaultCTA);
+      } else {
+        cancelCTAUpdate();
+        updatePageAttachment({
+          ctaText: _ctaText ? _ctaText : defaultCTA,
+        });
+      }
+    },
+    [_ctaText, cancelCTAUpdate, defaultCTA, updatePageAttachment]
+  );
+
   return (
     <SimplePanel
       name="pageAttachment"
       title={__('Page Attachment', 'web-stories')}
     >
       <LinkInput
-        description={__(
```

Comment:
Is this description unnecessary now?

Reply:
It's not visible anymore in the figma, maybe this should be a label somewhere though

Reply:
probs just for a11y